### PR TITLE
Add support for Create/Edit Domain motions

### PR DESCRIPTION
**Pull request by ArmandoGraterol**
This PR should support creating motions that add or edit domains by storing them in the DB.

CDapp counterpart: https://github.com/JoinColony/colonyCDapp/pull/379

### CODE REVIEW

File: src/handlers/motions/motionCreated/motionCreated.ts [hunk 1]
```diff
@@ -24,6 +25,11 @@ export default async (event: ContractEvent): Promise<void> => {
         await writeMintTokensMotionToDB(event, parsedAction);
         break;
       }
+      case ColonyOperations.AddDomain:
+      case ColonyOperations.EditDomain: {
+        await writeManageDomainMotionToDB(event, parsedAction);
+        break;
+      }
```
Comment (rdig) [inline src/handlers/motions/motionCreated/motionCreated.ts:32]: Will this overwrite the domain metadata entry in the case of a domain edit ?

**Review by rdig — APPROVED** [review]
Looks good, nothing stands out

File: src/handlers/motions/motionCreated/motionCreated.ts (hunk 1 above)
Comment (ArmandoGraterol) [reply src/handlers/motions/motionCreated/motionCreated.ts:32]: It shouldn't, this function's only purpose is to create a new `ColonyAction` entry

**Review by willm30 — APPROVED** [review]
LGTM. Is there more to come, e.g. connecting the domain created when finalizing the motion to the metadata?

**Comment by ArmandoGraterol** [comment]
> LGTM. Is there more to come, e.g. connecting the domain created when finalizing the motion to the metadata?

Yeah, I'll create an issue for that

**Pull request merged**